easybuild/easyconfigs/u/UDA/UDA-2.9.0-intel-compilers-2023.2.1.eb

```
@@ -0,0 +1,64 @@
+easyblock = 'CMakeNinja'
+
+name = 'UDA'
+version = '2.9.0'
+
+homepage = 'https://git.iter.org/projects/IMAS/repos/uda'
+description = """Universal Data Access"""
+
+toolchain = {'name': 'intel-compilers', 'version': '2023.2.1'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ukaea/UDA/archive/refs/tags']
+sources = ['%(version)s.zip']
+checksums = ['50b0dae7eb8b02345fb743c6c0d700bc76719a5cfc2222f4514914b79180e0dd']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('Ninja', '1.11.1'),
+    ('libMemcached', '1.0.18'),
+]
+
+dependencies = [
+    ('libxml2', '2.11.5'),
+    ('libtirpc', '1.3.4'),
+    ('Boost', '1.83.0'),
+    ('fmt', '10.2.1'),
+    ('spdlog', '1.12.0'),
+    ('CapnProto', '1.0.1.1'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+]
+
+separate_build_dir = True
+
+build_type = 'RelWithDebInfo'
+
+max_parallel = 1
+
+configopts = '-DBUILD_SHARED_LIBS=ON '
+configopts += '-DBOOST_ROOT=${EBROOTBOOST} '
+configopts += '-DENABLE_CAPNP=ON '
+configopts += '-DSSLAUTHENTICATION=ON '
+
+# Specify UDA hostname and port
+
+# modextravars = {
+#    'UDA_HOST': '',
+#    'UDA_PORT': '',
+# }
+
+sanity_check_paths = {
+    'files': ['bin/uda_server',
+              'bin/install_plugin',
+              'lib/libfatuda_client.%s' % SHLIB_EXT,
+              'lib/libuda_client.%s' % SHLIB_EXT,
+              'lib/libuda_plugins.%s' % SHLIB_EXT,
+              'lib/libuda_server.%s' % SHLIB_EXT, ],
```

```suggestion
```suggestion
              f'lib/libfatuda_client.{SHLIB_EXT}',
              f'lib/libuda_client.{SHLIB_EXT}',
              f'lib/libuda_plugins.{SHLIB_EXT}',
              f'lib/libuda_server.{SHLIB_EXT}', ],
```
```